docs/source/getting_started/authoring_your_workflow.md

```diff
@@ -18,91 +18,85 @@ The tutorial aims to be an extension to the quickstart to help you better unders
+
+### How to work with LatchFiles and LatchDirs
+
+`LatchFile` and `LatchDir` are types built into the Latch SDK which allow users to use files and directories on Latch as inputs and outputs to workflows. They point to remote file locations in our user interface on [Latch Console](https://console.latch.bio/data) and implement the necessary operations to ensure data is pulled into the task environment and saved back into your data tab.
+
+Any time you read or resolve the path of a file or directory in a task, the data will be downloaded locally and will be accessible within the task. For example the line `local_file = Path(read1)` in the snippet below will download the file from Latch and return a path to the local file.
+
 
 ```python
+from latch.types import LatchFile, LatchOutputDir
+...
+
 @small_task
-def assembly_task(read1: LatchFile, read2: LatchFile) -> LatchFile:
-
-    # A reference to our output.
-    sam_file = Path("covid_assembly.sam").resolve()
-
-    _bowtie2_cmd = [
-        "bowtie2/bowtie2",
-        "--local",
-        "-x",
-        "wuhan",
-        "-1",
-        read1.local_path,
-        "-2",
-        read2.local_path,
-        "--very-sensitive-local",
-        "-S",
-        str(sam_file),
-    ]
-
-    subprocess.run(_bowtie2_cmd)
-
-    return LatchFile(str(sam_file), "latch:///covid_assembly.sam")
-```
+def assembly_task(
+    read1: LatchFile, read2: LatchFile, output_directory: LatchOutputDir
+) -> LatchFile:
 
-### How to work with LatchFiles
+    local_file = Path(read1)
+    ...
+```
 
-`LatchFile` refers to a remote file location on
-[Latch Console](https://console.latch.bio/data).
+Sometimes, a python task does not directly access a file/directory, but uses it implicitly; for example, through a subprocessed shell command. We can ensure that the file's data is present by either calling `Path` on the file or directory, or by accessing the `local_path` attribute of the LatchFile or LatchDir. This will download the data into the task environment and return back a path to the local file or directory.
 
-To make the file available to the local task's environment, you have to call `local_path` property like so:
 ```python
-_bowtie2_cmd = [
-        "bowtie2/bowtie2",
-        "--local",
-        "-x",
-        "wuhan",
-        "-1",
-        read1.local_path,
-        "-2",
-        read2.local_path,
-        "--very-sensitive-local",
-        "-S",
-        str(sam_file),
-    ]
+bowtie2_cmd = [
+    "bowtie2/bowtie2",
+    "--local",
+    "--very-sensitive-local",
+    "-x",
+    "wuhan",
+    "-1",
+    read1.local_path,
+    "-2",
+    read2.local_path,
+    "-S",
+    str(sam_file),
+]
 ```
 
-Once the subprocess finishes executing the command above, it creates an output file (`str(sam_file)`).
-
-To make the SAM file available to Latch Console, you have to upload it using the following return statement:
+A file or directory can be outputted to the Latch platform as below:
```

Comment:
very confusing. explain that returning a `LatchFile`/`LatchDir` from the task/workflow will upload it to the specified remote path